snappy/desktop_test.go

```diff
@@ -124,58 +128,58 @@ Icon=/my/basedir/meep
 }
 
 func (s *SnapTestSuite) TestDesktopFileSanitizeFiltersExec(c *C) {
-	m, err := parseSnapYamlData([]byte(`
```

Comment:
Yay for getting rid of `parseSnapYamlData(s, hasConfig)` this bit was always a bit terrible, happy to see it gone.